build.sbt

```diff
@@ -15,11 +17,32 @@ ThisBuild / publishAsOSSProject := true
 lazy val publishTestsSettings = Seq(
   Test / packageBin / publishArtifact := true)
 
+lazy val quasarVersion = Def.setting[String](
+  managedVersions.value("precog-quasar"))
+
+val specs2Version = "4.8.3"
+
 lazy val root = project
   .in(file("."))
   .settings(noPublishSettings)
   .aggregate(core)
 
 lazy val core = project
   .in(file("core"))
-  .settings(name := "quasar-datasource-kafka")
+  .settings(addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1"))
```

Comment:
`sbt-precog` includes this, so it doesn't need to be added here

Reply:
Curious, I thought I had added it because it wasn't working for me. Maybe I just messed up something, then fixed it, and this plugin was just a coincidence.

Reply:
Or maybe it's part of the g8 template.